homeworks/src/homework18/TestClassNoAfter.java

```diff
@@ -0,0 +1,52 @@
+package homework18;
+
+public class TestClassNoAfter {
+    @BeforeSuite
+    public void MethodBeforeSuiteOneTwoAfter(){
+        System.out.println("MethodBeforeSuiteTwoAfter");
+    }
+
+    @Test
```

Comment:
You may add priority as a parameter to `@Test` annotation. Is have sense `@Priority` without `@Test`?